File-Formats/PDF/Convert-HTML-To-PDF/features.md

```diff
@@ -1834,3 +1834,171 @@ document.Close(True)
 {% endhighlight %}
 
 {% endtabs %}
+
+## Inject Custom CSS
+
+The Blink rendering engine provides the support for inject the custom CSS to be applied in HTML or URL before rendering to PDF. 
+
+{% tabs %}
+
+{% highlight c# tabtitle="C# [Cross-platform]" %}
+
+//Initialize HTML to PDF converter.
+HtmlToPdfConverter htmlConverter = new HtmlToPdfConverter();
+BlinkConverterSettings blinkConverterSettings = new BlinkConverterSettings();
+
+//Set Blink viewport size.
+blinkConverterSettings.ViewPortSize = new Size(1280, 0);
+blinkConverterSettings.Margin.All = 30;
+
+//Set custom CSS
+blinkConverterSettings.Css = "body {\r\n background-color: red; \r\n}";
+
+htmlConverter.ConverterSettings = blinkConverterSettings;
+//Convert URL to PDF document.
+PdfDocument document = htmlConverter.Convert(https://www.syncfusion.com);
+
+//Create a filestream.
+FileStream fileStream = new FileStream("Test.pdf", FileMode.CreateNew, FileAccess.ReadWrite);
+//Save and close the PDF document.
+document.Save(fileStream);
+document.Close(true);
+
+
+{% endhighlight %}
+
+{% highlight vb.net tabtitle="VB.NET [Windows-specific]" %}
+
+'Initialize HTML to PDF converter.
+Dim htmlConverter As HtmlToPdfConverter = New HtmlToPdfConverter()
+Dim blinkConverterSettings As BlinkConverterSettings = New BlinkConverterSettings()
+
+'Set Blink viewport size.
+blinkConverterSettings.ViewPortSize = New Size(1280, 0)
+blinkConverterSettings.Margin.All = 30
+
+'Set custom CSS
+blinkConverterSettings.Css = "body {\r\n background-color: red; \r\n}"
+
+htmlConverter.ConverterSettings = blinkConverterSettings
+'Convert URL to PDF document.
+Dim document As PdfDocument = htmlConverter.Convert(https://www.syncfusion.com)
+
+'Create a filestream.
+Dim fileStream As FileStream = New FileStream("Test.pdf", FileMode.CreateNew, FileAccess.ReadWrite)
+'Save and close the PDF document.
+document.Save(fileStream)
+document.Close(true)
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Inject Custom JavaScript
+
+The Blink rendering engine provides the support for inject the custom JavaScript to be applied in HTML or URL before rendering to PDF.
```

Comment:
Use this - "The Blink rendering engine offers support for injecting custom JavaScript to be applied to HTML or a URL before rendering it into a PDF document using the 'JavaScript' property of the 'BlinkConverterSettings' class."

Reply:
Provide API link for BlinkConverterSettings class